doc/classes/AnimatedSprite2D.xml

```diff
@@ -17,13 +19,14 @@
 			<param index="0" name="anim" type="StringName" default="&amp;&quot;&quot;" />
 			<param index="1" name="backwards" type="bool" default="false" />
 			<description>
-				Plays the animation named [param anim]. If no [param anim] is provided, the current animation is played. If [code]backwards[/code] is [code]true[/code], the animation will be played in reverse.
+				Plays the animation named [param anim]. If no [param anim] is provided, the current animation is played. If [param backwards] is [code]true[/code], the animation is played in reverse.
 			</description>
 		</method>
 		<method name="stop">
 			<return type="void" />
 			<description>
-				Stops the current animation (does not reset the frame counter).
+				Stops the current animation to the current [member frame].
+				[b]Note:[/b] This method also resets the frame's enlapsed time. If this behaviour is undesired, consider setting [member speed_scale] to [code]0[/code], instead.
```

Comment:
Typo and I think US English is preferred (thus "behavior").
```suggestion
				[b]Note:[/b] This method resets the current frame's elapsed time. If this behavior is undesired, consider setting [member speed_scale] to [code]0[/code], instead.
```

(if applied, the same for other stops)